library/src/lib/localizator-editor/localization-editor-item/localization-editor-item.component.html

```diff
@@ -0,0 +1,16 @@
+<li>
+    <div class="fd-input-group fd-input-group--after"
+         [ngClass]="{'fd-input-group--compact' : compact}"
+    >
```

Comment:
heavily dislike this style, I know it comes with vscode though. Put > on the previous line pls